lux-lib/src/operations/run_lua.rs

```diff
@@ -33,39 +38,68 @@ pub enum RunLuaError {
     },
     #[error(transparent)]
     Paths(#[from] PathsError),
+
+    #[error(transparent)]
+    Tree(#[from] TreeError),
 }
 
-pub async fn run_lua(
-    root: &Path,
-    tree: &Tree,
+#[derive(Builder)]
+#[builder(start_fn = new, finish_fn(name = run, vis = "pub"))]
+pub struct RunLuaBuilder<'a> {
```

Comment:
quibble: For consistency with our other builders, this should be called `RunLua`, not `RunLuaBuilder` (see https://github.com/nvim-neorocks/lux/pull/774#discussion_r2133863686)